stage-0-proposals.md

```diff
@@ -19,10 +19,12 @@ Stage 0 proposals are either
 |   | [Function expression decorators](https://goo.gl/8MmCMG)                                                                                  | Igor Minar                      | 0     |
 |   | [Zones](https://github.com/domenic/zones) ([spec](https://domenic.github.io/zones/))                                                     | Domenic Denicola & Miško Hevery | 0     |
 |   | [Updates to Tail Calls to include an explicit syntactic opt-in](https://github.com/tc39/proposal-ptc-syntax)                             | Brian Terlson & Eric Faust      | 0     |
-| 🚀 | [Unicode property escapes `\p{…}` and `\P{…}` in regular expressions](https://github.com/mathiasbynens/es-regexp-unicode-property-escapes) | Mathias Bynens, Daniel Ehrenberg, Brian Terlson   | 0     |
-| 🚀?| [Nested `import` declarations](https://github.com/tc39/ecma262/pull/646)                                                            | Ben Newman, Meteor Development Group | 0     |
-| 🚀 | [`s` (`dotAll`) flag for regular expressions](https://github.com/mathiasbynens/es-regexp-dotall-flag)                                    | Mathias Bynens, Brian Terlson   | 0     |
-|   | [WHATWG URL](https://github.com/jasnell/proposal-url)                                                 | James M Snell    | 0     |
+|   | [Nested `import` declarations](https://github.com/tc39/ecma262/pull/646)                                                                 | Ben Newman, Meteor Development Group | 0 |
```

Comment:
ah, that might be the case. if you'd rebased instead of merged master this wouldn't have occurred (in the PR, prior to merging)